webapp/sass/flamegraph.scss

```diff
@@ -0,0 +1,18 @@
+@use 'sass:math';
+@use 'variables' as *;
+@use 'mixins/outline' as *;
+
+@import '~sanitize.css';
+@import '~sanitize.css/forms.css';
+@import '~sanitize.css/typography.css';
```

Comment:
Won't these affect the user's global CSS as well?

Reply:
Theoretically it might affect user's global styles, but "cascading level" of these styles would be much lower than user's global styles, so I'm not sure that it can broke/change something.

>sanitize.css is a CSS library that provides consistent, cross-browser default styling of HTML elements alongside useful defaults. [[from here](https://csstools.github.io/sanitize.css/)]

Reply:
>but "cascading level" of these styles would be much lower than user's global styles,
I don't get this part. What if the user uses this other library https://github.com/resetercss/reseter.css for example, wouldn't our css rules (coming from sanitize.css) conflict with theirs?

Reply:
Here's an example

i modified `stories/Flamegraph.stories.tsx`, imagine this is the end user code
to have an `input` with red background (don't ask)
![image](https://user-images.githubusercontent.com/6951209/167896263-c822c1fa-f1e1-49cf-b3ec-1f246dd0fdc3.png)
![image](https://user-images.githubusercontent.com/6951209/167898438-34b5de46-8311-48fe-862e-6e00d5c1c0e2.png)

![image](https://user-images.githubusercontent.com/6951209/167898493-a2f8b6f3-795a-47dd-a931-0dd9f454b2d7.png)
then i imported the flamegraph.css
![image](https://user-images.githubusercontent.com/6951209/167898679-d8822a38-6630-4957-91e4-cb4061bdc399.png)

Now my input is not red anymore!
![image](https://user-images.githubusercontent.com/6951209/167898643-db3b7567-d84c-40b2-9211-bb0990f717e8.png)